integration_tests/pages/refer/programmeHistoryDetails.ts

```diff
@@ -7,6 +8,34 @@ export default class ProgrammeHistoryDetailsPage extends Page {
     super('Add Accredited Programme details', { accessibilityRules: { 'aria-allowed-attr': { enabled: false } } })
   }
 
+  inputOutcomeDetail(value: string) {
+    cy.get('textarea[id="outcomeDetail"]').type(value)
+  }
+
+  inputSettingLocation(value: string) {
+    cy.get('input[id="communityLocation"]').type(value)
+  }
```

Comment:
Do we need two methods: one for community and one for custody?

Reply:
Just the one for now, will rename to `inputCommunityLocation` though.